angular/projects/spark-angular/src/lib/components/sprk-alert/sprk-alert.component.ts

```diff
@@ -21,28 +21,36 @@ import { Component, Input } from '@angular/core';
       </div>
 
       <button
-        *ngIf="dismissible !== false"
+        *ngIf="isAlertDismissible !== false"
         class="sprk-c-Alert__icon sprk-c-Alert__icon--dismiss"
         type="button"
         title="Dismiss"
         (click)="alertDismiss($event)"
       >
         <sprk-icon
-          iconType="close"
+          iconName="{{ dismissIconName }}"
           additionalClasses="sprk-c-Icon--filled-current-color sprk-c-Icon--stroke-current-color"
           aria-hidden="true"
         ></sprk-icon>
       </button>
     </div>
   `,
 })
-export class SprkAlertComponent {
+export class SprkAlertComponent implements OnInit {
   /**
    * The type of Alert variant to render.
-   * 	Can be `info`, `fail`, or `success`.
+   * Can be `info`, `fail`, or `success`.
    */
   @Input()
-  alertType: string;
+  variant: 'success' | 'fail' | 'info';
+  /**
+   * Deprecated - Use `variant` instead.
+   * The type of Alert variant to render.
+   * Can be `info`, `fail`, or `success`.
+   */
+  // TODO: Remove `alertType` and update variant default to be info.
```

Comment:
Can you work with Steph to get an issue in for this so we don't lose track? Can put issue # in comment too.

Reply:
Got it added!